docs/src/doc/decisions-and-ADRs/adr/0008-css-scoping.stories.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Documentation|Decisions and ADRs/ADRs/8) CSS scoping" />
+
+# 8) CSS scoping
+
+Date: 2020-07-24
+
+## Status
+
+Accepted
+
+## Context
+
+CSS authors need to take care to not cause undesired global side effects by creating overly greedy selectors (e.g. styling certain tags on the whole page instead of just the desired ones).
+
+Vue offers a built-in solution for this challenge called [component-scoped CSS](https://vuejs.org/v2/guide/comparison.html#Component-Scoped-CSS). It [adds a data attribute](https://vue-loader.vuejs.org/guide/scoped-css.html#scoped-css) to all component's elements and extends the CSS selectors to match them.
```

Comment:
Before reading https://vue-loader.vuejs.org/guide/scoped-css.html#child-component-root-elements I never really thought about the "leak into child components" aspect (only about the way _up_), but I guess with our way of very precise classes this is covered just as well.

Reply:
I'm also a little scared about version mismatches between potentially server-side rendered markup and css served by ResourceLoader. That's adventurous enough without scope hashes that are out of our control.